ui/src/app/kvm/views/KVMDetails/LxdResources/LxdResources.tsx

```diff
@@ -18,16 +18,19 @@ const LxdResources = ({ id }: Props): JSX.Element => {
   const pod = useSelector((state: RootState) =>
     podSelectors.getById(state, Number(id))
   );
-
   useWindowTitle(`LXD resources ${pod?.name || ""}`);
 
   if (!!pod) {
     return (
       <>
         <Strip className="u-no-padding--top" shallow>
+          <h4 className="u-sv1">Overall</h4>
```

Comment:
I've moved the headings out of the card components to make it easier to put the card in other places (e.g. in virsh details)